src/ansys/fluent/core/codegen/settingsgen.py

```diff
@@ -195,8 +195,9 @@ def _get_unique_name(name):
 def _write_function_stub(name, data, s_stub):
     s_stub.write(f"    def {name}(self")
     for arg_name in data["argument_names"]:
-        arg_type = _arg_type_strings[data["child_classes"][arg_name]["bases"][0]]
-        s_stub.write(f", {arg_name}: {arg_type}")
+        arg_type = data["child_classes"][arg_name]["bases"][0]
+        py_arg_type = _arg_type_strings.get(arg_type, "Any")
```

Comment:
We can specify TypedDict as type-hint for dictionaries with known keys, will check..